services/harmony/app/markdown/capabilities.md

```
@@ -4,49 +4,70 @@ Returns information related to what Harmony operations are supported for a given
 
 ##### <a name="query-parameters"></a> Query Parameters
 Either `collectionId` or `shortName` must be provided.
-| parameter    | description                                                                                    |
-|--------------|------------------------------------------------------------------------------------------------|
-| collectionId | Concept id of the collection to retrieve capabilities for                                      |
-| shortName    | Short name of the collection to retrieve capabilities for                                      |
-| version      | (optional) The version of the capabilities result format, currently 1, 2, and 3 are supported. |
+| parameter    | description |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)